<launch>
    <!-- <param name="/use_sim_time" value="true"/> -->
    <param name="/GRIDROI/link_shp"     type="str" value="$(find multiobj_tracker)/MOT_HDMAP/LINK_minho_V4.shp"/>
    <param name="/GRIDROI/link_layer"   type="str" value="LINK_minho_V4"/>
    <param name="/GRIDROI/lane_shp"     type="str" value="$(find multiobj_tracker)/MOT_HDMAP/LANE_minho_V4.shp"/>
    <param name="/GRIDROI/lane_layer"   type="str" value="LANE_minho_V4"/>
    <!-- <param name="/GRIDROI/lane_shp"     type="str" value="$(find multiobj_tracker)/MOT_HDMAP/LANE_minho_V2.shp"/>
    <param name="/GRIDROI/lane_layer"   type="str" value="LANE_minho_V2"/> -->

    <rosparam command="load" file="$(find multiobj_tracker)/config/mot.yaml"/>

    <!-- <node pkg="multiobj_tracker" type="gridROI_generator" name="gridROI_generator" output= "screen"/> -->
    <node pkg="multiobj_tracker" type="lidar_detector" name="lidar_detector" output= "screen"/>
    <!-- <node pkg="multiobj_tracker" type="lidarSort.py" name="lidar_sort" output= "screen"/> -->
    <!-- <node pkg="multiobj_tracker" type="lidar_classifier" name="lidar_classifier" output= "screen"/> -->

    <node pkg="multiobj_tracker" type="gridROI_generator" name="gridROI_generator"/>
    <!-- <node pkg="multiobj_tracker" type="lidar_detector" name="lidar_detector" /> -->
    <node pkg="multiobj_tracker" type="lidarSort.py" name="lidar_sort"/>
    <node pkg="multiobj_tracker" type="lidar_classifier" name="lidar_classifier"/>

    <!-- <node type="rviz" name="rviz" pkg="rviz" args="-d $(find multiobj_tracker)/rviz/multiobj_tracker.rviz" /> -->

</launch>
